sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationManagerImpl.kt

```diff
@@ -0,0 +1,238 @@
+package com.mapbox.maps.viewannotation
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewTreeObserver
+import android.widget.FrameLayout
+import androidx.annotation.LayoutRes
+import androidx.asynclayoutinflater.view.AsyncLayoutInflater
+import com.mapbox.maps.*
+import com.mapbox.maps.plugin.ViewPlugin
+import java.util.concurrent.ConcurrentHashMap
+
+internal class ViewAnnotationManagerImpl(
+  private val mapView: MapView,
+  private val mapboxMap: MapboxMap = mapView.getMapboxMap()
+) : ViewAnnotationManager, ViewAnnotationPositionsListener {
+
+  init {
+    mapView.requestDisallowInterceptTouchEvent(false)
+    mapboxMap.setViewAnnotationPositionsUpdateListener(this)
+  }
+
+  private val asyncInflater by lazy {
+    try {
+      AsyncLayoutInflater(mapView.context)
+    } catch (e: NoClassDefFoundError) {
+      throw RuntimeException(
+        "Please add https://mvnrepository.com/artifact/androidx.asynclayoutinflater/asynclayoutinflater/1.0.0 dependency " +
```

Comment:
Instead of dealing with possible crashes I'd suggest to let user pass `AsyncInflater` as a property or pass it to the method directly, e.g. 
```addViewAnnotationAsync(
  ...
  asyncInflater : AsyncLayoutInflater
)
```

Reply:
PR is in https://github.com/mapbox/mapbox-maps-android/pull/844